File: Day_1/Java/Day_1.java

/*     
 DAY 1 : Take an input character from the user and check whether
             the given input is a vowel or consonant.
-----------------
 Input
 A
 Output
 Vowel
 --------------------
 Input
 m
 Output
 Consonant
 -----------------
 Input
 3
 Output
 Invalid Input
 ------------------
 */
import java.util.Scanner;

public class Day_1 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        System.out.print("Please Enter an Alphabet : ");

        // for taking only fist char from user enter value
        char ch = sc.next().charAt(0);

        if (ch == 'A' || ch == 'E' || ch == 'I' || ch == 'O' || ch == 'U' || ch == 'a' || ch == 'e' || ch == 'i'
                || ch == 'o' || ch == 'u') {
            System.out.println("Vowel");
        } else if ((ch >= 'A' && ch <= 'Z') || (ch >= 'a' && ch <= 'z')) {
            System.out.println("Consonent");
        } else {
            System.out.println("Invalid Input");
        }
    }
}

File: Day_01/C++/Day_1.cpp
#include<iostream>
using namespace std;

int main() {
    char  c;

    cout << "Enter an alphabet: ";
    cin >> c;
cout<<c;    
    if(c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u'||c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U')
    {
         cout << c << " is a vowel.";
    }
    else if(isalpha(c))//isaplha method is to check wheter value is alphabet or not 
    {
         cout << c << " is a consonant.";
    }
    else
    {
    cout << c << " is not a valid input.";
    }
    return 0;
}
